### PR TITLE
make readme more readable

### PR DESCRIPTION
**Pull request by jhaals**

### CODE REVIEW

**Review by Rugvip — CHANGES_REQUESTED** [review]
UNACCEPTABLE

**Comment by jhaals** [comment]
NO!

File: README.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ repository.
   - [ ] Automate the move to the "Changes Requested" column.
   - [ ] Custom logic for adding PRs to the board, skip PRs that do not have DCO,
         but add them once they do.
+        
+
```
Comment (Rugvip) [inline README.md:30]: BUT SPACE

**Review by Rugvip — CHANGES_REQUESTED** [review]
UNACCEPTABLE

File: README.md (hunk 1 above)
Comment (jhaals) [reply README.md:30]: WE NEED IT!

Comment (benjdlambert) [reply README.md:30]: NO SPACE

**Comment by Rugvip** [comment]
SPACE

File: README.md (hunk 1 above)
Comment (Rugvip) [reply README.md:30]: SO MUCH SPACE

Comment (Rugvip) [reply README.md:30]: 👾

**Pull request closed without merging**